addon/i3dio/node_classes/shape.py

```diff
@@ -206,11 +206,12 @@ def process_subset(self, mesh, material_name: str, triangle_offset: int = 0):
             for loop_index in triangle.loops:
                 blender_vertex = mesh.vertices[mesh.loops[loop_index].vertex_index]
 
+
                 # Add vertex color
                 vertex_color = None
                 if len(mesh.vertex_colors):
-                    # Get the color from the active layer, since only one vertex color layer is supported in GE
-                    vertex_color = mesh.vertex_colors.active.data[loop_index].color
+                    # Get the color from the first layer, since only one vertex color layer is supported in GE
+                    vertex_color = mesh.vertex_colors[0].data[loop_index].color

```

Comment:
What is the reason for this change? Has the API changed?

Reply:
I have got an error where active returned None, but the model had color attribute. Maybe it would be better idea then using first layer, just check whether active is not None